lib/yaml_elixir.ex

```diff
@@ -52,8 +52,8 @@ defmodule YamlElixir do
     {:yamerl_exception, [{_, _, message, _, _, :file_open_failure, _, _}]} ->
       {:error, %YamlElixir.FileNotFoundError{message: List.to_string(message)}}
 
-    {:yamerl_exception, [{_, _, message, _, _, :no_matching_anchor, _, _}]} ->
-      {:error, %YamlElixir.ParsingError{message: List.to_string(message)}}
+    {:yamerl_exception, [error | _]} ->
+      {:error, YamlElixir.ParsingError.from_yamerl(error)}
```

Comment:
Annotation: According to the `yamerl` documentation it is possible to return multiple errors.  For now lets just take the first one.